draft/spec/index.html

```diff
@@ -861,7 +861,32 @@ <h2>Logs Directory</h2>
                 wrong, while others may wish to only store a log entry if an intervention was required.
             </p>
         </blockquote>
+    </section>
 
+    <section id="object-extensions">
+        <h2>Object Extensions</h2>
+        <p>
+            The base directory of an OCFL Object MAY contain a directory named <code>extensions</code> for the purposes
+            of extending the functionality of an OCFL Object. The <code>extensions</code> directory MUST NOT contain
+            any files, and no sub-directories other than extension sub-directories. Extension sub-directories SHOULD be
+            named according to a registered extension name. The specific structure and function of the extension, as
+            well as a declaration of the registered extension name MUST be defined in one of the following locations:
+        </p>
+        <ul>
+          <li>
+            The <a href="https://ocfl.github.io/extensions/">OCFL Extensions repository</a>
+          </li>
+          <li>
+            The Storage Root, as a plain text document directly in the Storage Root
+          </li>
+        </ul>
+        <blockquote class="informative">
+          <p>
+            Non-normative note: Extension sub-directories should use the same name as a registered extension
+            in order to both avoid the possiblity of an extension sub-directory colliding with the name of another
+            registered extension as well as to facilitate the recognition of extensions by OCFL consumer clients.
```

Comment:
remove 'consumer'? We don't really define what that is.

Reply:
agree to drop "consumer"